doc/_static/theme_overrides.css

```diff
@@ -20,5 +20,25 @@
     .option {
         width: 1px !important;
     }
+
+
+    .hide {
+        display: none;
+    }
+    .hide + label ~ div {
+        display: none;
+    }
+    .hide + label {
+        display: block;
```

Comment:
I propose to add some minor styling here: font size, background color, cursor.
Otherwise it is not clear that this is interactive element.
Also its font is even smaller than that of the normal text.